src/main/kotlin/no/nav/syfo/prediksjon/PrediksjonOutputQueries.kt

```diff
@@ -0,0 +1,37 @@
+package no.nav.syfo.prediksjon
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import no.nav.syfo.database.DatabaseInterface
+import no.nav.syfo.database.toList
+import no.nav.syfo.domain.AktorId
+import no.nav.syfo.domain.Fodselsnummer
+import java.sql.ResultSet
+import java.time.ZoneOffset
+
+const val queryHentPrediksjon = "SELECT * FROM prediksjon_output WHERE fnr = ?"
```

Comment:
For at dette skal fungere bra må det lages en indeks på fnr-kolonnen i prediksjon_output.